packages/edit-site/src/components/sidebar/global-styles-v2/index.js

```diff
@@ -0,0 +1,192 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	CardHeader,
+	ContextSystemProvider,
+	Heading,
+	Navigator,
+	NavigatorScreen,
+	NavigatorScreens,
+	View,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ColorPickerModal,
+	Inspector,
+	URLSync,
+	useInitialPath,
+} from './components';
+import { ANIMATION_SPEED } from './constants';
+import {
+	ColorsElementScreen,
+	ColorsPaletteScreen,
+	ColorsScreen,
+	GlobalStylesScreen,
+	TypographyElementScreen,
+	TypographyScreen,
+} from './screens';
+import { AppProvider } from './state';
+
+/**
+ * This is the "root" Component (or app) for Global Styles.
+ */
+
+export default {
+	title: 'Examples/WIP/GlobalStylesSidebarV2',
+};
+
+/**
+ * These are the "routes" that bind navigation paths with components for the
+ * indivudal screens.
+ */
+const screens = [
+	{
+		component: GlobalStylesScreen,
+		path: '/',
+		title: 'Global Styles',
+	},
+	{
+		component: ColorsScreen,
+		path: '/colors',
+		title: 'Colors',
+	},
+	{
+		component: ColorsPaletteScreen,
+		path: '/colors/palette',
+		title: 'Palette',
+	},
+	{
+		component: ColorsElementScreen,
+		path: '/colors/elements/:id',
+		title: 'Color Element',
+	},
+	{
+		component: TypographyScreen,
+		path: '/typography',
+		title: 'Typography',
+	},
+	{
+		component: TypographyElementScreen,
+		path: '/typography/elements/:id',
+		title: 'Typography Element',
+	},
+];
+
+/**
+ * GlobalStylesHeaer and Sidebar can largely be ignored.
+ * These components exist to scaffold/bootstrap the Global Styles Sidebar
+ * app into a simluated Gutenberg UI environment.
+ */
+
+const GlobalStylesHeader = () => {
+	return (
+		<CardHeader>
+			<Heading size={ 5 }>Global Styles</Heading>
+		</CardHeader>
+	);
+};
+
+const Sidebar = ( { children } ) => {
+	return (
+		<ContextSystemProvider
+			value={ {
+				Grid: { gap: 2 },
+				Icon: { size: 16 },
+			} }
+		>
+			<View
+				css={ `
+					width: 280px;
+					position: absolute;
+					height: 100vh;
+					top: 0;
+					right: 0;
+					/* @TODO: Update this to properly reflect the colorDivider color */
+					border-left: 1px solid #444;
```

Comment:
This was hardcoded to prevent the need from having to access the internal `COLOR` constant from the component utils. Needs to retrieve the color value dynamically.